rpc-client-api/src/response.rs

```diff
@@ -306,6 +306,8 @@ pub struct RpcContactInfo {
     pub feature_set: Option<u32>,
     /// Shred version
     pub shred_version: Option<u16>,
+    /// First 4 bytes of the sha1 commit hash
+    pub commit: Option<u32>,
```

Comment:
maybe we should serialize to a hex string, since that's the common display format for a git commit hash?

Reply:
+1, I like that idea